src/pages/iou/ReceiptSelector/index.native.js

```diff
@@ -89,8 +89,7 @@ function getImagePickerOptions(type) {
 }
 
 function ReceiptSelector({route, report, iou, transactionID, isInTabNavigator}) {
-    const devices = useCameraDevices('wide-angle-camera');
-    const device = devices.back;
+    const device = useCameraDevice('back');
```

Comment:
Shouldn't we also enable wide-angle-camera too?

Reply:
Nope, that's already handled by default :)